src/main/target/NAZE/initialisation.c

```diff
@@ -33,12 +33,15 @@ void targetBusInit(void)
     #endif
 
     if (hardwareRevision != NAZE32_SP) {
-        i2cInit(I2C_DEVICE);
         serialRemovePort(SERIAL_PORT_SOFTSERIAL2);
     } else {
         if (!doesConfigurationUsePort(SERIAL_PORT_USART3)) {
             serialRemovePort(SERIAL_PORT_USART3);
-            i2cInit(I2C_DEVICE);
-        }		
+        }
     }
-}
\ No newline at end of file
+
+    // Regardless of the board revision, initialize I2C as
+    // pre-configured or according to the current resource assignment.
+
+    i2cInitAll();
```

Comment:
Call    `i2cInitAll(i2cPinConfig());` - will need to include `include "config/config_master.h"`